config/settings.yml

```diff
@@ -1433,10 +1433,21 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
   jwt:
     public_key_path: modules/meb_api/spec/fixtures/dgi_public_test.pem
     private_key_path: modules/meb_api/spec/fixtures/dgi_private_test.pem
+  vye:
+    jwt:
+      # May not need the public path
```

Comment:
I would confirm whether or not you need the public path. If you don't need it then don't use it

Reply:
done